docs/source/configure.rst

```diff
@@ -73,6 +73,12 @@ To change the default password for the Quipucords server, use the following step
 
 **TIP:** You can also enter the local or remote URL (as applicable) for the Quipucords server in a browser window to verify that the Quipucords server is responding.
 
+Changing the Default Database Management System
```

Comment:
Let's remove this change in doc since @abaiken covers this in her PR (see https://github.com/quipucords/quipucords/pull/1439)